packages/i/ipaddress/xmake.lua

```diff
@@ -0,0 +1,95 @@
+package("ipaddress", function()
+    set_homepage("https://vladimirshaleev.github.io/ipaddress/")
+    set_description("A library for working and manipulating IPv4/IPv6 addresses and networks")
+    set_license("MIT")
+    set_kind("library", {
+        headeronly = true
+    })
+
+    add_urls("https://github.com/VladimirShaleev/ipaddress/archive/refs/tags/$(version).tar.gz",
+        "https://github.com/VladimirShaleev/ipaddress.git")
+
+    add_versions("v1.1.0", "e5084d83ebd712210882eb6dac14ed1b9b71584dede523b35c6181e0a06375f1")
+
+    add_configs("noexcept", {
+        description = "Disable handling cpp exception for",
+        default = false,
+        type = "boolean"
+    })
```

Comment:
use single line for each config

Reply:
Is there any format tool for xmake? This file is formatted by the Lua extension os vscode.

Reply:
no tool, you can see other packages

Reply:
please do not any format tool